StackOverflowQuickLaunch/source.extension.vsixmanifest

```diff
@@ -11,11 +11,9 @@
     <PreviewImage>Resources\Preview_200x.png</PreviewImage>
   </Metadata>
   <Installation InstalledByMsi="false">
-    <InstallationTarget Version="11.0" Id="Microsoft.VisualStudio.Pro" />
-    <InstallationTarget Version="12.0" Id="Microsoft.VisualStudio.Community" />
-    <InstallationTarget Version="14.0" Id="Microsoft.VisualStudio.Community" />
-    <InstallationTarget Version="15.0" Id="Microsoft.VisualStudio.Community" />
-    <InstallationTarget Version="16.0" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[12.0, 17.0]"/>

```

Comment:
Actually, the range values are "up to but not including 17.0". I have tested it using 16.0, and it will not work with updated copies of 2019. I have added another commit to standardise the layout, and include 11.0 - 17.0 (VS 2012 -2019)

https://devblogs.microsoft.com/visualstudio/how-to-upgrade-extensions-to-support-visual-studio-2019/